hms-app/src/components/PitchTimer.tsx

```diff
@@ -150,11 +203,19 @@ function PitchTimer(time: PitchTimerProps) {
           renderer={renderer}
         />
       </Group>
-      <Progress
-        value={progress}
-        color={isCompleted() ? 'red' : 'gray'}
-        mt={10}
-      />
+      {timeLeft > 0 && timeLeft < 4000 && (
+        <div
+          className={`${classes.fullscreenOverlay} ${
+            timeLeft > 1000 ? classes.bgOrange : classes.bgRed
+          }`}
+        >
+          <Center>
+            <span className={classes.fullscreenCounter}>
+              {getSeconds(timeLeft)}
```

Comment:
these seems a bit odd, we pass in the remaining seconds to the function, then convert them (and the minutes) to a millisecond value and then here we convert that back to seconds :thinking:

But this could be > 60 right? so you could have like 450 seconds

in fact do we EVER need millisecond precision here? can't we just store remaining seconds?

Reply:
ah no, it will count down from 4 seconds, so never more than 60.

Reply:
true, but I could refactor it a bit. Just need the millis to be added to the Date.now()